packages/mystjs/docs/Makefile

```diff
@@ -7,13 +7,13 @@ fix-default-js:
 
 build:
 	cd ..; npm run build
-	cp ../dist/index.umd.min.js _static/myst.min.js
+	cp ../dist/index.cjs.js _static/myst.min.js
```

Comment:
Why is the `cjs` build used here, instead of the browser build?

Reply:
Ah, yeah, I flipped that back and forth and didn't check that. Will fix now.